src/ops/watch.rs

```diff
@@ -19,10 +43,19 @@ pub fn main(project: Project) -> OpResult {
     };
 
     for msg in rx {
-        println!("{:#?}", msg);
+        print_build_message(msg);
+        let _ = std::io::stdout().flush();
```

Comment:
```suggestion
        let _ = std::io::stdout().flush();
```

Should this be part of `print_build_message`?